src/amber/cli/commands/new.cr

```diff
@@ -22,6 +22,12 @@ module Amber::CLI
 
       def run
         Amber::CLI.color = !options.no_color?
+        full_path_name = File.join(`pwd`.sub("\n", ""), args.name)
+        if full_path_name.includes?(" ")
+          puts "Error:  Path and project name can't contain a space. Underscores are fine though."
```

Comment:
Extra space in `Error: Path`. Replace puts with `CLI.logger.error`

Also maybe instead of `Underscores are fine though.` write something like `Use underscores or dash instead of spaces.`

```
CLI.logger.error "Error: Path and project name can't contain a space." 
CLI.logger.error "Replace spaces with underscores or dashes." (Replace/Use)
```

Reply:
Good suggestions.